src/main/java/ch/njol/skript/util/chat/ChatMessages.java

```diff
@@ -506,10 +506,16 @@ public static String stripStyles(String text) {
 		}
 		String plain = sb.toString();
 		
-		// To be extra safe, strip <, >, § and &; protects against bugs in parser
-		if (Utils.HEX_SUPPORTED) // Strip '§x'
-			plain = plain.replace("§x", "");
-		plain = plain.replace("<", "").replace(">", "").replace("§", "").replace("&", "");
+		// To be extra safe, strip valid <, >, §, &; protects against bugs in parser
+		if (Utils.HEX_SUPPORTED) // Strip '§x', '&x'
+			plain = plain.replace("[§&]x", "");
+		/*
+			1st Replace: strips json
+			2nd Replace: strips colors using <color name> (NOTE: This regex expression replaces any two words surrounded by <> such as <any text>)
+			3rd Replace: strips colors & or §
+		*/
+		plain = plain.replaceAll("<(link|(open |)url|(run |)command|cmd|suggest command|sgt|tooltip|show text|ttp|insertion|insert|ins):(.*?)>", "$2").replaceAll("<\\w+? ?\\w+?>", "").replaceAll("(?i)[&§][0-9a-folkrnm]", ""); // (NOTE: These values are changable in message config, so this better be improved later)
```

Comment:
It might be a bit annoying to handle, but it would be ideal to only replace color names surrounded by `<` and `>`.

Reply:
I will try to get the color names from the config and replace them here, no problem.